client/my-sites/plans/jetpack-plans/product-lightbox/index.tsx

```diff
@@ -187,11 +219,52 @@ const ProductLightbox: React.FC< Props > = ( {
 									/>
 								</div>
 							) }
-							<PaymentPlan
-								isMultiSiteIncompatible={ isMultiSiteIncompatible }
-								siteId={ siteId }
-								product={ product }
-							/>
+							{ isSocialProduct ? (
```

Comment:
I'm wondering if we want to turn this into something more generic, since we might add more products with multiple plan options in the future.

What do you think about adding some kind of check that checks if the product has multiple plan options, rather than checking if it's a Social product?